js/main.js

```diff
@@ -251,6 +276,81 @@ function directorInit() {
   });
 
 }
+// OVERLAY IMAGE
+var overlayImage = {
+
+  player: $('#image-overlay-player'),
+
+  overlay: $('#image-overlay'),
+  overlayInner: $('#image-overlay-inner'),
+
+  timeout: 0,
+
+  load: function(image, postIndex) { 
+    var _this = this;
+
+    image.attr('width', '100%').removeAttr('height');
+
+    _this.overlay.fadeIn(fastAnimationSpeed).data('now-playing', postIndex);
+    _this.player.html(image);
+
+    // use fixHeight after animations
+      _this.fixHeight();
+    
+    $(window).resize(function() {
+      this.timeout = setTimeout(function() {
+        _this.fixHeight();
+      }, 50);
+    });
+
+
+  },
+
+  next: function () {
+    var nowPlayingIndex = this.overlay.data('now-playing'),
+      nextIndex = nowPlayingIndex + 1;
+    if (directorStillsLenght > nextIndex) {
+      var image = directorStills.eq(nextIndex).children('img').clone();
+      this.load(image, nextIndex);
+    } else {
+      var image = directorStills.eq(0).children('img').clone();
+      this.load(image, 0);
+    }
+  },
+
+  prev: function () {
+    var nowPlayingIndex = this.overlay.data('now-playing'),
+      prevousIndex = nowPlayingIndex - 1;
+    if (prevousIndex === -1) {
+      var image = directorStills.eq(directorStillsLenght - 1).children('img').clone();
+      this.load(image, directorStillsLenght - 1);
+    } else {
+      var image = directorStills.eq(prevousIndex).children('img').clone();
+      this.load(image, prevousIndex);
+    }
+  },
+
+  close: function () {
+    this.overlay.fadeOut(fastAnimationSpeed).data('now-playing', '');
+    this.player.html('');
+  },
+
+  fixHeight: function () {
```

Comment:
couldn't all this functionality be fixed for an image just with css? or is it not worth bothering?